pkg/retry/retry.go

```diff
@@ -109,5 +109,11 @@ func isErrnoRetryable(e error) bool {
 	case syscall.ECONNREFUSED, syscall.EINTR, syscall.EAGAIN, syscall.EBUSY, syscall.ENETDOWN, syscall.ENETUNREACH, syscall.ENETRESET, syscall.ECONNABORTED, syscall.ECONNRESET, syscall.ETIMEDOUT, syscall.EHOSTDOWN, syscall.EHOSTUNREACH:
 		return true
 	}
+	switch err := e.(type) { //nolint
```

Comment:
Why would this go into `is`**`Errno`**`Retryable` and not into existing top-level `isErrnoRetryable` type switch?

Reply:
… into the existing top-level `isRetyrable` type switch

Reply:
Cause I am an idiot.... Fixed.